pkg/storage/postgres/postgres.go

```diff
@@ -33,6 +33,7 @@ type Postgres struct {
 	logger                 logger.Logger
 	maxTuplesPerWriteField int
 	maxTypesPerModelField  int
+	currentTime            storage.TimeProvider
```

Comment:
```suggestion
	timeProvider            storage.TimeProvider
```

Reply:
Obsolete as we don't have the time provider anymore